airflow/executors/base_executor.py

```diff
@@ -122,7 +122,7 @@ class BaseExecutor(LoggingMixin):
     change_sensor_mode_to_reschedule: bool = False
     serve_logs: bool = False
 
-    job_id: None | int | str = None
+    job_id: int | str | None = None
```

Comment:
Btw. Maybe we could avoid this change to make it providers-only change ?

Reply:
sure